src/index.ts

```diff
@@ -0,0 +1,11 @@
+import * as express from 'express';
+import { App } from './App';
+import { Sequelize } from 'sequelize';
+
+const APPLICATION_PORT = 8080;
+
+// TODO: set user and password by environment variables
+const app = new App(express(), new Sequelize('postgres://postgres:yourpassword@localhost:5432/mydb'));
```

Comment:
You could change the postgres URL to a variable as well... maybe even have it in a separate file. But i guess it will come in near future and I see that you have a TODO. A suggestion is that maybe you could receive the whole ConnectionString instead of only user and pass from env.

When I'm developing an application I pass an object to the ORM (I believe Sequelize does it as well) so in such case you could pass it granularly every variable in a better way.
I also usually initialize the ORM in a separate file as well... for now it's fine. It's up to you to update it in a separate file to initialize the Sequelize object.

Reply:
I already started to work on the next task on the separate branch, I will address your comments on the next PR.